src/components/widgets/Widget.tsx

```diff
@@ -5,13 +5,19 @@ import { useHistory } from 'react-router-dom';
 interface Props {
   title: string;
   subtitle?: string;
+  goBackRoute?: string;
```

Comment:
it's great that this comes from an upper level component. Next step we can consider just passing a callback, i.e. passing `goBack: Function<void>` , implemented in the parent by 
```js
const goBack = () => {
    history.push(goBackRoute);
  };
```